src/shared-components/dropdown/desktopDropdown.js

```diff
@@ -44,11 +56,12 @@ const DesktopDropdown = ({
               selected={value === optionValue}
               disabled={disabled}
               onClick={onOptionClick}
+              // eslint-disable-next-line react/jsx-props-no-spreading
               {...rest}
```

Comment:
`...rest` will now pass in `label` since we're no longer destructuring it above. Is that intentional? 

It looks like it was an oversight that the destructured `label` from option wasn't used below in place of `{option.label}`. I think we might more likely avoid regression by continuing to destructure the `label` in the map callback and instead replace `{optional.label}` with just `{label}`.

Reply:
good call!